README.md

```diff
@@ -109,6 +109,7 @@ If you're having issues with the packaged executable or using this tool in gener
 | Spanish (Español)                           | [enriiquee](https://github.com/enriiquee)     | UI                                   |
 | German (Deutsch)                            | [hebens](https://github.com/hebens)           | UI (partial)                         |
 | Russian (Русский)                           | Oleg                                          | UI                                   |
+| Thai (ภาษาไทย)                              | [teerut26](https://github.com/teerut26)       | UI (99%)                             |
```

Comment:
Can we have `UI (partial)` instead of `UI (99%)` to keep consistent. If you have a translation for "Output", then we can have just `UI`.